### PR TITLE
Quiz questions

### PR DESCRIPTION
**Pull request by GuptaEesh**
> Added questions screen
questionscreen

> Added results screen
resultsPage

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@import url('/quizQuestions/questionStyle.css');
+@import url('/quizQuestions/result.css');
+
+:root{
+    --spacing-one:1rem;
+    --spacing-two:2rem;
+    --spacing-three:4rem;
```
Comment (shivampandey0) [inline index.css]: `--spacing-three: 4rem;`
wouldn't it be hard to remember the variables? having `three` in name and linking to `4 rem`.
why not use it like this:
`--spacing-4: 4rem;`
`--spacing-6: 6rem;`
`--spacing-8: 8rem;`
...

**Review by shivampandey0 — COMMENTED** [review]
Looks good
- CSS variables are used for colors and units
- structured code with meaningful class names

Added one suggestion.

File: index.css (hunk 1 above)
Comment (GuptaEesh) [reply index.css]: it's in multiple of two and enumerable from one.

Comment (shivampandey0) [reply index.css]: Talked with the dev and got the rationale behind the naming convention. 
Good to merge

**Review by AnuragSahu11 — APPROVED** [review]
Good work, the UI is clean and beautiful, you can go ahead and merge

File: index.css (hunk 1 above)
Comment (GuptaEesh) [reply index.css]: I have changed the naming as the changes suggested were good. Thankyou 😀

**Review by saadahmadhere — APPROVED** [review]
Looks great to me.

**Pull request merged**